src/pages/_app.tsx

```diff
@@ -55,6 +55,7 @@ function MyApp({ Component, pageProps, theme, adobeApiKey }: TProps) {
           <ErrorBoundary level="top">
             <Head>
               <link rel="icon" href={favicon} />
+              <link rel="stylesheet" href="https://use.typekit.net/ifw7ubm.css" />
```

Comment:
Let's catch up on this - this will load the font into every app. Something I would like to avoid if at all possible

Reply:
If we got to this URL there is some CSS we could add into the MCF `.scss` file in the theme directory:
https://use.typekit.net/ifw7ubm.css

Then we can remove this from the app head.